packages/config/src/api/site_info.ts

```diff
@@ -1,8 +1,21 @@
+import { NetlifyAPI } from 'netlify'
 import fetch from 'node-fetch'
 
 import { getEnvelope } from '../env/envelope.js'
 import { throwUserError } from '../error.js'
 import { ERROR_CALL_TO_ACTION } from '../log/messages.js'
+import { IntegrationResponse } from '../types/api.js'
+import { ModeOption, TestOptions } from '../types/options.js'
+
+type GetSiteInfoOpts = {
+  siteId: string
+  mode: ModeOption
+  siteFeatureFlagPrefix: string
+  offline?: boolean
+  api?: NetlifyAPI
+  featureFlags?: Record<string, boolean>
+  testOpts?: TestOptions
+}
 
 // Retrieve Netlify Site information, if available.
```

Comment:
NIT: can you convert that into a proper jsdoc comment so that it can be seen via the intellisense?


```
/**
 * Retrieve Netlify Site information, if available.
 * Used to retrieve local build environment variables and UI build settings.
 * Requires knowing the `siteId` and having the access `token`.
 * Silently ignore API errors. For example the network connection might be down,
 * but local builds should still work regardless.
 */